src/core/solvers/ddp.cpp

```diff
@@ -124,10 +125,12 @@ bool SolverDDP::solve(const std::vector<Eigen::VectorXd>& init_xs, const std::ve
 }
 
 void SolverDDP::computeDirection(const bool recalcDiff) {
+  START_PROFILER("SolverDDP::computeDirection");
```

Comment:
It would be nice to include the BoxDDP solver

Reply:
If the BoxDDP solver inherits and uses this method, then it will be profiled. However, it will appear with the name "SolverDDP::computeDirection". The only way around this is to overload computeDirection in the BoxDDP solver

Reply:
You're right for `computeDirection`, but not for the other points.

I will mark this conversation as resolved :)